shotover-proxy/src/message/mod.rs

```diff
@@ -345,26 +350,33 @@ impl Value {
             match spec.col_type.id {
                 ColType::Ascii => Value::Strings(decode_ascii(actual_bytes).unwrap()),
                 ColType::Bigint => Value::Integer(decode_bigint(actual_bytes).unwrap()),
-                ColType::Blob => Value::Bytes(Bytes::copy_from_slice(actual_bytes)),
                 ColType::Boolean => Value::Boolean(decode_boolean(actual_bytes).unwrap()),
-                ColType::Counter => Value::Integer(decode_int(actual_bytes).unwrap() as i64),
-                ColType::Decimal => unimplemented!("We dont have a decimal type yet"),
+                ColType::Counter | ColType::Int => {
+                    Value::Integer(decode_int(actual_bytes).unwrap() as i64)
+                }
+                ColType::Decimal => todo!(),
```

Comment:
Do we make another `Value` variant for Decimal or store it in the float one?

Reply:
Decimal can not be losslessly stored as a float.
So I think our only option is to create a Decimal variant.

Reply:
In that case we can use https://docs.rs/bigdecimal/latest/bigdecimal/index.html

Reply:
Yup we need a lossless Decimal type (probably a bigint as well)